src/metrics/models.py

```diff
@@ -55,6 +55,45 @@ def remove_one_download(self):
         self.save()
 
 
+def object_types():
+    return (
+        ('book', 'Book'),
+        ('article', 'Article'),
+    )
+
+
+class ForwardLink(models.Model):
+    article = models.ForeignKey('submission.Article', blank=True, null=True)
+    doi = models.CharField(max_length=255)
+    object_type = models.CharField(max_length=10, choices=object_types())
+    year = models.CharField(max_length=5)
```

Comment:
year shouldn't be an instance of  `models.CharField` I don't think :P

Reply:
I had this as a positive integer field but some of the data from crossref sucks so covering arses by using a charfield.

Reply:
What other kinds of 'year' have you seen? I think we should instead not store the year if the format is not valid and log it

Reply:
What else can be returned other than year? I think we should not store the year if the format is not valid.